pkgs/by-name/sy/syshud/package.nix

```diff
@@ -14,18 +14,22 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "syshud";
-  version = "0-unstable-2024-09-26";
+  version = "0-unstable-2024-11-04";
 
   src = fetchFromGitHub {
     owner = "System64fumo";
     repo = "syshud";
-    rev = "0b6e4958d8ea66b54bc67f0b5aa005fa3eaa4f6f";
-    hash = "sha256-cyE7ZxesAmnepW7kI6b9Gp7R/v+yG2//EQAj4/X6c+c=";
+    rev = "157b725a3f29d67f16c25fb3062b62ad6fec4e15";
+    hash = "sha256-q04xYOdnnyUyiFc72Gzk65fWzQgYSOACPUXIk7kvIP8=";
   };
 
   postPatch = ''
     substituteInPlace Makefile \
-      --replace-fail 'pkg-config' ''${PKG_CONFIG}
+      --replace-fail pkg-config ''${PKG_CONFIG}
```

Comment:
`''${PKG_CONFIG}` is intentional: the `''` escapes the `$` so that `${PKG_CONFIG}` is evaluated by bash, otherwise nix would parse this as string interpolation.

the single quotes in `'pkg-config'` aren't needed, removed those because it felt inconsistent.

Reply:
thank you for explanation, it's first time i have seen  `''` inside `''`, but it works as you have described !